PortfolioApp/Model/UsersModels/CompanyModels.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+struct Company: Decodable {
```

Comment:
The file should name as struct name. Don't invent custom names. Files name should be equal to model name